doc/news/changes/minor/20240821Marquis

```diff
@@ -0,0 +1,8 @@
+Fixed: Previously, FiniteElement::has_generalized_support_points() and 
+FiniteElement::has_face_support_points() returned false for the FE_Nothing element. Now, FE_Nothing::has_generalized_support_points() and 
+FE_Nothing::has_face_support_points() correctly return true, as the empty 
+arrays returned by FE_Nothing::get_generalized_support_points() and FE_Nothing::get_unit_face_support_points() accurately describe the support 
+points of the element(i.e., they don't have any, as there are no
+degrees of freedom).
+<br>
+(Oreste Marquis, 2024/08/21)
```

Comment:
github says that you need an endline here. (In fact, every file should always end in a newline.)